### ISSUE TITLE
Small mistake in usePresenceChannel documentation

### ISSUE THREAD

**Issue by Kozie1337** [mayteio/use-pusher#28]
Hey, I enjoy using the package so far but I noticed a mistake with the "usePresenceChannel" documentation.

Instead of
```
    {Object.entries(members)
        // filter self from members
        .filter([id] => id !== myID)
        // map them to a list
        .map([id, info] => (
          <li key={id}>{info.name}</li>
        ))
      }
```
It should be 
```
    {Object.entries(members)
        // filter self from members
        .filter(id => id !== myID)
        // map them to a list
        .map((info, id ) => (               // info 1st parameter, id 2nd
          <li key={id}>{info.name}</li>
        ))
      }
```

Correct me if I'm wrong, and maybe it's just on my end, but it took me some time to figure out  😄

**Comment by mayteio** [comment]
Hey @Kozie1337, glad you like the package and thanks for helping out! Do you have a codesandbox reproducing this issue?

Object.entries returns an array of key value pairs (also in an array) thus the `[id, info]` deconstruction - [see MDN docs on Object.entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries). 

I’m curious to see what it was returning for you.

**Comment by Kozie1337** [comment]
I was using it inside my project and I did nothing more than pasting the example code inside a (new) React component.
When I paste the example code I get a bunch of errors like 
``` ',' expected.ts(1005)```
and  ```Unexpected token. Did you mean `{'>'}` or `&gt;`?ts(1382)```

The .filter returns a 2D Array for me just as you said,  but the id (```(id, info) =>...```) returns the info, and the info returns the id. Maybe I mixed up something within my pusher-auth process regarding this.

And I can't even try out what it returns the way it is in the documentation (```[id,info]=>...```) because of all the errors. 

But in general, try calling the ```.map([a,b]=>())```structure on any 2D Array inside any code editor will retrieve errors.
